qstack/regression/regression.py

```diff
@@ -2,7 +2,18 @@
 
 import numpy as np
 import scipy
-from sklearn.model_selection import train_test_split
+
+try:
+    from sklearn.model_selection import train_test_split
+except ImportError:
+    print("""
+
+ERROR: cannot import scikit-learn. Have you installed qstack with the \"regression\" option?\n\n
+(for instance, with `pip install qstack[regression] or `pip install qstack[all]``)
+
+""")
+    raise
+
```

Comment:
A bunch of files in qstack.regression use sklearn, do you think it's possible to complain not here (and not in each of them) but e.g. in qstack.regression.__init__?